src/components/pages/faq/index.js

```diff
@@ -11,13 +11,13 @@ Guesstimate is great at making estimates of things that aren't certain.  You can
 The most popular model on Guesstimate is [How Long it Takes to Get Ready for Preschool](/models/314).  We also recommend checking out Ozzie Gooen's models [here](/users/1).
 
 ## Are there private models?
-Not yet.  Please sign up for an account to get on our mailing list, or follow us [on Twitter](https://twitter.com/getguesstimate) to be notified once there are.  We expect private models in the next 1-6 weeks.
+Yes. Check out our [pricing page](/pricing) for more information.
```

Comment:
Yes. -> Yes!

At the end say 'Let us know what other features you want in our premium accounts.'